Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
```

Comment:
This package is probably wrong because it is declaring the root folder where we usually put our packages, which is not related to the packages. Please check it out: https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html